test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2710,6 +2710,29 @@ def forward(self, input, batch1, batch2, alpha, beta):
         model = MyModule()
         self.run_test(model, (x, batch1, batch2, alpha, beta))
 
+    def test_linspace(self):
```

Comment:
if this test is for opset 11 shouldn't it contain this line @skipIfUnsupportedMinOpsetVersion(11)?

Reply:
Yeah, missed this. Thanks!